### PR TITLE
Genre retireval

### PR DESCRIPTION
**Pull request by amazing-username**
Feature for retrieving the genre name if the genre from the ID3v2_frame_text_content data member is in the ID3 format like this `(7)` which would be Hip-Hop. The supported genres can be found [here](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=5&cad=rja&uact=8&ved=2ahUKEwi9yM7J6OfiAhWFnOAKHdBeB0cQygQwBHoECAAQBg&url=http%3A%2F%2Fid3.org%2Fid3v2.3.0%23Appendix_A_-_Genre_List_from_ID3v1&usg=AOvVaw2z_NeAff1zfwDeTcHnudH8). The function to parse the genre number is 
```C 
void genre_num_string(char* genre_data, char* dest);
```

The genre_data parameter would be the ID3v2_frame_text_context->data variable and the dest parameter would be where the appropriate genre would be assigned to.

### CODE REVIEW

File: src/frame.c [hunk 1]
```diff
@@ -65,11 +65,12 @@ ID3v2_frame_text_content* parse_text_frame_content(ID3v2_frame* frame)
     {
         return NULL;
     }
-    
+
     content = new_text_content(frame->size);
     content->encoding = frame->data[0];
-    content->size = frame->size - ID3_FRAME_ENCODING;
```
Comment (larsbs) [inline src/frame.c]: Why this change?

Comment (amazing-username) [reply src/frame.c]: I changed that to fix an issue I have been having with mangled text. However, that solved the issue but not for some songs that I was testing with but not all. I'll revert it back to the way it was before.
